.gitlab-ci.yml

```diff
@@ -47,10 +73,40 @@ build:docker:ui:
   variables:
     DOCKER_REPOSITORY: mendersoftware/mantra-ui
     DOCKER_DIR: ui
+  after_script:
+    - cp image.tar image-ui.tar
+  artifacts:
+    expire_in: 2w
+    paths:
+      - image.tar
+      - image-ui.tar
   rules:
     - changes:
         - ui/**/*
 
+test:composition:
+  stage: acceptance_tests
+  image: tiangolo/docker-with-compose
+  needs:
+    - job: build:docker
+      artifacts: true
+    - job: build:docker:ui
+      artifacts: true
+  services:
+    - docker:20.10.8-dind
+  before_script:
+    - docker import image-api.tar mendersoftware/mantra-api:pr
+    - docker import image-ui.tar mendersoftware/mantra-ui:pr
+    - apk add --no-cache python3
+    - pip3 install tox
+    - echo $'[api]\nbase_url = http://test:test@docker/api' > backend/tetra-test.conf
```

Comment:
Q:
What's the `$` good for here?

Reply:
line breaks! not sure if there is a better way, but putting the line break in the ci file doesn't play nice with gitlab, this however does 👌